deployment/changesets/cs_add_new_fee_token.go

```diff
@@ -0,0 +1,109 @@
+)
+
+type NewFeeTokenConfig struct {
+	SuiChainSelector  uint64
+	FeeTokensToRemove []string
+	FeeTokensToAdd    []string // should be the objectID
+
+	// update price
+	SourceUsdPerToken []*big.Int
+
+	// premium multiplier wei per eth
+	PremiumMultiplierWeiPerEth []uint64
+}
+
+// ConnectSuiToEVM connects sui chain with EVM
+type NewFeeToken struct{}
+
+var _ cldf.ChangeSetV2[NewFeeTokenConfig] = NewFeeToken{}
+
+// Apply implements deployment.ChangeSetV2.
+func (d NewFeeToken) Apply(e cldf.Environment, config NewFeeTokenConfig) (cldf.ChangesetOutput, error) {
+	ab := cldf.NewMemoryAddressBook()
+	state, err := deployment.LoadOnchainStatesui(e)
+	if err != nil {
+		return cldf.ChangesetOutput{}, err
+	}
+
+	seqReports := make([]operations.Report[any, any], 0)
+
+	suiChains := e.BlockChains.SuiChains()
+	suiChain := suiChains[config.SuiChainSelector]
+
+	deps := sui_ops.OpTxDeps{
+		Client: suiChain.Client,
+		Signer: suiChain.Signer,
+		GetCallOpts: func() *bind.CallOpts {
+			b := uint64(400_000_000)
+			return &bind.CallOpts{
+				WaitForExecution: true,
+				GasBudget:        &b,
+			}
+		},
+		SuiRPC: suiChain.URL,
+	}
+
+	// Run ApplyFeeTokenUpdate Operation
+	applyFeeTokenUpdateOP, err := operations.ExecuteOperation(e.OperationsBundle, ccipops.FeeQuoterApplyFeeTokenUpdatesOp, deps, ccipops.FeeQuoterApplyFeeTokenUpdatesInput{
+		CCIPPackageId:     state[suiChain.Selector].CCIPAddress,
+		StateObjectId:     state[suiChain.Selector].CCIPObjectRef,
+		OwnerCapObjectId:  state[suiChain.Selector].CCIPOwnerCapObjectId,
+		FeeTokensToRemove: config.FeeTokensToRemove,
+		FeeTokensToAdd:    config.FeeTokensToAdd,
+	})
+	if err != nil {
+		return cldf.ChangesetOutput{}, fmt.Errorf("failed to register receiver for Sui chain %d: %w", config.SuiChainSelector, err)
+	}
+
+	seqReports = append(seqReports, []operations.Report[any, any]{applyFeeTokenUpdateOP.ToGenericReport()}...)
+
+	// Run ApplyPremiumMultiplier
+	applyPremiumMultiplierOP, err := operations.ExecuteOperation(e.OperationsBundle, ccipops.FeeQuoterApplyPremiumMultiplierWeiPerEthUpdatesOp, deps, ccipops.FeeQuoterApplyPremiumMultiplierWeiPerEthUpdatesInput{
+		CCIPPackageId:              state[suiChain.Selector].CCIPAddress,
+		StateObjectId:              state[suiChain.Selector].CCIPObjectRef,
+		OwnerCapObjectId:           state[suiChain.Selector].CCIPOwnerCapObjectId,
+		Tokens:                     config.FeeTokensToAdd,
+		PremiumMultiplierWeiPerEth: config.PremiumMultiplierWeiPerEth,
+	})
+	if err != nil {
+		return cldf.ChangesetOutput{}, fmt.Errorf("failed to register receiver for Sui chain %d: %w", config.SuiChainSelector, err)
+	}
+
+	seqReports = append(seqReports, []operations.Report[any, any]{applyPremiumMultiplierOP.ToGenericReport()}...)
+
+	// Run UpdateTokenPrice Operation
+	updateTokenPriceOP, err := operations.ExecuteOperation(e.OperationsBundle, ccipops.FeeQuoterUpdateTokenPricesWithOwnerCapOp, deps, ccipops.FeeQuoterUpdateTokenPricesInput{
```

Comment:
I don't think we need this after CCIP is already working properly with Arbitrum feeds.